ssl/ssl_test.cc

```diff
@@ -12112,10 +12112,10 @@ TEST(SSLTest, SSLFileTests) {
   using ScopedFILE = std::unique_ptr<FILE, FileCloser>;
 
 #if defined(OPENSSL_WINDOWS)
-  char rsa_pem_filename[L_tmpnam];
-  char ecdsa_pem_filename[L_tmpnam];
-  ASSERT_EQ(tmpnam_s(rsa_pem_filename, sizeof(rsa_pem_filename)), 0);
-  ASSERT_EQ(tmpnam_s(ecdsa_pem_filename, sizeof(ecdsa_pem_filename)), 0);
+  char rsa_pem_filename[FILENAME_MAX];
+  char ecdsa_pem_filename[FILENAME_MAX];
+  ASSERT_EQ(tmpnam_s(rsa_pem_filename, FILENAME_MAX), 0);
+  ASSERT_EQ(tmpnam_s(ecdsa_pem_filename, FILENAME_MAX), 0);
```

Comment:
Interesting, why does this need to change to `FILENAME_MAX`? Does `sizeof(rsa_pem_filename)` return the wrong length?

Reply:
The mingw header files were using a value of 12, which was too small and causing the function to fail.

Reply:
Microsoft's documentation says that the parameter is the size of the buffer "in characters", which should be the same but when it was failing I decided to be pedantic with it.